lib/katello/tasks/pulp3_post_migration_check.rake

```diff
@@ -0,0 +1,33 @@
+require File.expand_path("../engine", File.dirname(__FILE__))
+
+namespace :katello do
+  desc "Runs a post Pulp3 migration check for supported content types."
+  task :pulp3_post_migration_check => :environment do
+    repository_types = Katello::Pulp3::Migration::REPOSITORY_TYPES
+
+    repository_types.each do |type|
+      repositories = Katello::Repository.with_type(type)
+        .where('version_href is NULL OR remote_href is NULL OR publication_href is NULL')
+      if repositories.any?
+        $stderr.print("ERROR: at least one #{type} repository record has a NULL value for remote_href, version_href, publication_href\n")
```

Comment:
I'm still hitting this error, I think you need to add your Docker filter before it.